### ISSUE TITLE
Gota problem with double signing

### ISSUE THREAD

**Issue by peterzhangbo** [cosmos/cosmos#100]
What is the meaning of Double Signing exactly?
Double signing: If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A

And why be punished with it?

**Comment by rigelrozanski** [comment]
Interesting, assuming you got that from here https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md

I think this is confusing and needs to be reworded/made more explicit. Within the current definition, I think `chain A` and `chain B` are referring to two different "sections" so-to-speak of the _same_ blockchain. In a poorly connected node network, there may only be a small number of nodes which connect two different groups of nodes which compose the entire validator set - in this case, if this "small number of nodes" were to suddenly go offline, the network would be considered _partitioned_. Likewise, malicious groups of nodes may try to use their position within a network to broadcast different transactions to different sides of the blockchain for nefarious reasons. 

double signing is _not_ in reference to signing two different chains - that would be acceptable behaviour as the chains are distinct (but possible sharing a validator set, these are complex but potentially allowable arrangements). Within the cosmos double signing typically refers two signing and broadcasting two _different_ pieces of information (for instance a proposed block) for the _same_ height for the _same_ chain. 

Hope this helps. Going to rename this issue to bring clarity to this point in the docs

**Comment by peterzhangbo** [comment]
@rigelrozanski U got me,thank you.

look forward to getting more information in updating docs.

**Comment by rigelrozanski** [comment]
yeah many of the docs are not in cosmos-sdk aka https://github.com/cosmos/cosmos-sdk/tree/develop/docs
aka https://cosmos.network/docs/ not this repo even, not sure what's going to become of this repo yet

**Comment by hsohaib** [comment]
This is to prevent the "nothing-At-stake" scenario, essentialy forcing the validator to choose one chain to prolong
